portal/src/main/webapp/faq.jsp

```diff
@@ -40,45 +40,41 @@
 
 <%@taglib prefix="t" tagdir="/WEB-INF/tags" %>
 
-<t:template title="<%= siteTitle %>" defaultRightColumn="true" fixedWidth="true">
+<t:template title="<%= siteTitle %>" defaultRightColumn="false" fixedWidth="true">
 
     <jsp:attribute name="head_area">
         <!-- js files: -->
-        <script type="text/javascript" src="js/lib/jquery.min.js?${appVersion}"></script>
-        <script type="text/javascript" src="js/lib/underscore-min.js?${appVersion}"></script>
-        <script type="text/javascript" src="js/lib/backbone-min.js?${appVersion}"></script>
-
-        <script type="text/javascript" src="js/lib/showdown.min.js?${appVersion}"></script>
-        <script type="text/javascript" src="js/lib/showdown-github.min.js?${appVersion}"></script>
-        <script type="text/javascript" src="js/src/url_based_content.js?${appVersion}"></script>
         <script>
         window.loadReactApp({ defaultRoute: 'blank' });
+        
+        var to;
+        
+        function setIframeHeight() {
```

Comment:
necessary to make iframe always approx fill the page without a scrollbar on browser portal